addons/web_editor/static/src/js/editor/image_processing.js

```diff
@@ -201,15 +203,17 @@ const glFilters = {
  * containing the result. This function does not modify the original image.
  *
  * @param {HTMLImageElement} img the image to which modifications are applied
+ * @param {Object} imageOptions a dictionary of the options to apply to the
+ * image
```

Comment:
Ok, but then what is `dataOptions` (which seems unused) ?

Reply:
I will add it on the method documentation :+1: